### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by JoshBriggs**
Added content to game section, tested and works well.

### CODE REVIEW

File: baw_game_a1.py [hunk 1]
```diff
@@ -29,18 +30,22 @@ def print_and_wait(wait_text):
 
 #Core Ideas
 #Player creates character's name
-char_name = input('What is your character\'s name?\n')
-print('Your character\'s name is ' + char_name)
-print_and_wait('Welcome to your adventure ' + char_name + '. Your destiny awaits!')
+player_name = input('What is your character\'s name?\n')
```
Comment (jameswalk3r) [inline baw_game_a1.py:33]: I like that change. Makes sense.

File: baw_game_a1.py [hunk 2]
```diff
@@ -29,18 +30,22 @@ def print_and_wait(wait_text):
...
+print('Your character\'s name is ' + player_name)
+print_and_wait('Welcome to your adventure ' + player_name + '. Your destiny awaits!')
+print (' ')
```
Comment (jameswalk3r) [inline baw_game_a1.py]: Why are we printing empty spaces? Is there a cleaner way to do this?

File: baw_game_a1.py [hunk 3]
```diff
@@ -67,7 +72,7 @@ def print_and_wait(wait_text):
 
 #Player selects class.
 print('What is your character\'s class?\n')
-c = int(input('Press 1 for Warrior, 2 for Mage, or 3 for Assassin.'))
+c = int(input('Press 1 for Warrior, 2 for Mage, or 3 for Assassin.\n'))
```
Comment (jameswalk3r) [inline baw_game_a1.py:75]: sweetness!

File: baw_game_a1.py [hunk 4]
```diff
@@ -78,5 +83,24 @@ def print_and_wait(wait_text):
...
+    print ('You have decided to press forward.')
+elif player_move == 2:
+    print ('You have decided to turn back.')
+elif player_move == 3:
+    print ('You have decided to explore the ruins further\n')
+    explore_choice = input ('Which way do you want to go?\n Left or Right?\n')
+    if str.lower(explore_choice) ==  'left':
+        print ('something')
+    elif str.lower(explore_choice) ==  'right':
+        print ('something else')
```
Comment (jameswalk3r) [inline baw_game_a1.py:102]: Boom!!!! Love it!

File: test.py [hunk 5]
```diff
@@ -27,13 +27,30 @@ def print_and_wait(wait_text):
...
-    print('fart')
-elif n == 2:
-    print('wet fart')
-elif n == 3:
-    print('shard')
-elif n == 4:
-    print('poo')
-else: print('you moron!')
\ No newline at end of file
+# n = int(input('give me a number, 1-4\n'))
```
Comment (jameswalk3r) [inline test.py:30]: What are all these changes? You didn't make any mention of this...

**Review by jameswalk3r — CHANGES_REQUESTED** [review]
Nice work Josh!! Have a question on the PR. Also, there is a silly space. Let's clean up that code and then we will get this into master.

File: test.py (hunk 5 above)
Comment (ghost) [reply test.py:30]: Made those lines into comments because I didn't need to test them anymore.

**Review by jameswalk3r — APPROVED** [review]

**Pull request merged**